README.md

```diff
@@ -60,6 +61,20 @@ Connector runtimes provided by the Community:
 |  GitHub Dispatch          |            -           |      [GitHub](https://github.com/camunda-community-hub/camunda-8-connector-github-dispatch)           |   [README](https://github.com/camunda-community-hub/camunda-8-connector-github-dispatch/blob/main/README.md)            |   Apache 2.0      |         [![](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)  |
 | OpenAI Moderation | - | [GitHub](https://github.com/MaxTru/camunda-connector-openai-moderation) | [README](https://github.com/MaxTru/camunda-connector-openai-moderation/blob/main/README.md) | MIT       |         [![](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)  |
 
+# Adding connectors
+
+You are able to add your connector in just a few steps:
+- Be inspired with a brilliant idea!
```

Comment:
maybe we can an option to only share the idea. Sth. Like this:
You want to develop it yourself and share it with the community?
Or do you just want to share your idea with the community? Link to feature request****

Reply:
Added a side note on a feature request.